### ISSUE TITLE
Accessibility on social sharing icons

### ISSUE THREAD

**Issue by ghost** [presscustomizr/hueman-addons#62]
social share icons links are missing aria labels or title tags and are therefore inaccessible to screenreaders.

https://secure.helpscout.net/conversation/1685600870/227706/
